README.md

```diff
@@ -9,3 +9,25 @@ It detects whether the container environment contains secret references (`secret
 1. It prefixes the target containers command with `<path/to/volume>/secrethub run --`.
 
 This project is based on and heavily inspired by [Berglas's Kubernetes Mutating Webhook](https://github.com/GoogleCloudPlatform/berglas/tree/v0.5.1/examples/kubernetes).
+
+## Deploy the Webhook
+
+The simplest method to deploy the webhook is in a serverless function. Below we outline the steps to take to deploy the webhook to a Google Cloud Function.
+We're also [working on](https://github.com/secrethub/secrethub-kubernetes-mutating-webhook/pull/2) a way to deploy the webhook in the Kubernetes cluster itself.
+
+You can deploy the webhook to a Google cloud function using the following steps:
+
+1. Deploy the webhook to a Google Cloud Function:
+```sh
+gcloud functions deploy secrethub-mutating-webhook --runtime go113 --trigger-http
```

Comment:
What if you get the question if you want to allow unauthenticated requests?

Reply:
I've allowed unauthenticated requests. Access to the function doesn't give you access to any secret data or resources, it just enables you to change a pod specification you have access to in a predefined way.

If you do want restricted access, I imagine using IAM or #2

Reply:
I'll add a note.

Reply:
done